src/dosarray_graphing_focus.sh

```diff
@@ -5,13 +5,26 @@
 #
 # Use of this source code is governed by the Apache 2.0 license; see LICENSE
 #
-# $ DOSARRAY_HIST_FOCUS=10 python ../generate_availability_chart.py '../../exp4/sl_60total_start10_last20_apache_event/c*.log' > availability.data
-# $ ../dosarray_graphing_focus.sh . '' 10 30
+# $ export DOSARRAY_VERSION="0.3"
```

Comment:
Not clear why this line matters.

Reply:
Its required by generate_availability_chart.py if dosarray_graphing_focus.sh is run separately

Reply:
Thanks for confirming -- could you add it as a comment to the `# export ...` line pls so it'll be clear in the code.